lib/datasource/dart/__snapshots__/index.spec.ts.snap

```diff
@@ -2,7 +2,6 @@
 
 exports[`datasource/dart getReleases processes real data 1`] = `
 Object {
-  "homepage": "https://github.com/flutter/plugins/tree/master/packages/shared_preferences",
```

Comment:
Deleted because it's github link which usually goes to `sourceUrl`. But it's already present for this fixture.